lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -660,6 +659,123 @@ def get(self):
             folderIndex[groups].items.append(attributeInfo.name)
         return self._attributeLayout
 
+class DisplayCustomDataControl(object):
```

Comment:
New custom control class which add the outliner color fields to the "Display" section.